components/aws/sagemaker/Dockerfile

```diff
@@ -17,8 +17,9 @@ RUN apt-get update -y && apt-get install --no-install-recommends -y -q ca-certif
 
 RUN easy_install pip
 
-RUN pip install boto3==1.9.130 sagemaker pathlib2
+RUN pip install boto3==1.9.169 sagemaker pathlib2 pyyaml==3.12
```

Comment:
Since it's a revision version change, I don't have concern. Better to run exiting workflows to make sure this boto version works for all previous jobs.

Reply:
I've run a pipeline using the existing components using this version of boto and they still work